lib/logstash/inputs/kafka.rb

```diff
@@ -36,10 +36,19 @@ class LogStash::Inputs::Kafka < LogStash::Inputs::Base
   # the same consumer group.
   config :group_id, :validate => :string, :default => 'logstash'
   # The topic to consume messages from
-  config :topic_id, :validate => :string, :required => true
+  config :topic_id, :validate => :string, :default => nil
```

Comment:
why do we have topic_id to be optional? what does it mean to have nil as default? Can we make this required?